test/unit/test_manifest_download.py

```diff
@@ -0,0 +1,86 @@
+import errno
+import os
+import shutil
+import tempfile
+import unittest
+
+import scandir
+from mock import patch
+
+from hca.dss import DSSClient
+
+
+def _touch_file(path):
+    try:
+        os.makedirs(os.path.split(path)[0])
+    except OSError as e:
+        if e.errno == errno.EEXIST:
+            pass
+    with open(path, 'w'):
+        pass
+
+
+def _fake_download(*args, **kwargs):
+    _touch_file(args[2])
+
+
+class TestManifestDownload(unittest.TestCase):
+    manifest = [
+        ['bundle_uuid', 'bundle_version', 'file_content_type', 'file_name', 'file_sha256', 'file_size', 'file_uuid', 'file_version', 'file_indexed'],
+        ['a_uuid', '1_version', 'somestuff', 'a_file_name', 'ad3fc1e4898e0bce096be5151964a81929dbd2a92bd5ed56a39a8e133053831d', '12', 'af_uuid', 'af_version', 'False'],
+        ['b_uuid', '1_version', 'somestuff', 'b_file_name', '8f35071eaeedd9d6f575a8b0f291daeac4c1dfdfa133b5c561232a00bf18c4b4', '2', 'bf_uuid', 'af_version', 'False'],
+        ['c_uuid', '1_version', 'somestuff', 'c_file_name', '8f3404db04bdede03e9128a4b48599d0ecde5b2e58ed9ce52ce84c3d54a3429c', '41', 'cf_uuid', 'af_version', 'False'],
+    ]
```

Comment:
```diff
@@ -25,12 +27,20 @@
 
 
 class TestManifestDownload(unittest.TestCase):
-    manifest = [
-        ['bundle_uuid', 'bundle_version', 'file_content_type', 'file_name', 'file_sha256', 'file_size', 'file_uuid', 'file_version', 'file_indexed'],
-        ['a_uuid', '1_version', 'somestuff', 'a_file_name', 'ad3fc1e4898e0bce096be5151964a81929dbd2a92bd5ed56a39a8e133053831d', '12', 'af_uuid', 'af_version', 'False'],
-        ['b_uuid', '1_version', 'somestuff', 'b_file_name', '8f35071eaeedd9d6f575a8b0f291daeac4c1dfdfa133b5c561232a00bf18c4b4', '2', 'bf_uuid', 'af_version', 'False'],
-        ['c_uuid', '1_version', 'somestuff', 'c_file_name', '8f3404db04bdede03e9128a4b48599d0ecde5b2e58ed9ce52ce84c3d54a3429c', '41', 'cf_uuid', 'af_version', 'False'],
-    ]
+    manifest = list(zip(
+        ('bundle_uuid', 'a_uuid', 'b_uuid', 'c_uuid'),
+        ('bundle_version', '1_version', '1_version', '1_version'),
+        ('file_content_type', 'somestuff', 'somestuff', 'somestuff'),
+        ('file_name', 'a_file_name', 'b_file_name', 'c_file_name'),
+        ('file_sha256',
+         'ad3fc1e4898e0bce096be5151964a81929dbd2a92bd5ed56a39a8e133053831d',
+         '8f35071eaeedd9d6f575a8b0f291daeac4c1dfdfa133b5c561232a00bf18c4b4',
+         '8f3404db04bdede03e9128a4b48599d0ecde5b2e58ed9ce52ce84c3d54a3429c'),
+        ('file_size', '12', '2', '41'),
+        ('file_uuid', 'af_uuid', 'bf_uuid', 'cf_uuid'),
+        ('file_version', 'af_version', 'af_version', 'af_version'),
+        ('file_indexed', 'False', 'False', 'False'),
+    ))
 
     def setUp(self):
         self.prev_wd = os.getcwd()
```

Reply:
I find that more readable than a 169 character line. Then again some people don't know what zip() does and might get confused. This is a suggestion.

Reply:
Ok, I see your point.

Reply:
Suggestion taken.